androidHyperskillApp/src/main/java/org/hyperskill/app/android/home/view/ui/fragment/HomeFragment.kt

```diff
@@ -207,4 +231,10 @@ class HomeFragment :
             )
         }
     }
+
+    private fun renderTopicsToDiscoverNext(state: TopicsToDiscoverNextFeature.State) {
+        viewBinding.homeTopicsToDiscoverNextTitle.isVisible =
+            state is TopicsToDiscoverNextFeature.State.Content
+        topicsToDiscoverNextDelegate.render(state)
```

Comment:
**Learn next** badge is only visible once it loaded -> disappears after navigation to any other screen and comes back to the Home Screen

[device-2023-02-03-120619.webm](https://user-images.githubusercontent.com/8586063/216517345-e02bd42f-114d-4058-a587-4f3276c68790.webm)